### ISSUE TITLE
Crash with coredump on dell xps 15 (linux, openGL)

### ISSUE THREAD

**Issue by maarzt** [scenerygraphics/scenery#150]
I get this error when trying to run SciView on my DELL XPS15. @kephale said these infos might actually be helpful to fix a bug. For me SciView sometimes works, and sometimes crashes and I get different error messages. So it's not reproducible.
```
#
#  SIGSEGV (0xb) at pc=0x00007fb549c27820, pid=7018, tid=0x00007fb488697700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_161-b12) (build 1.8.0_161-b12)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.161-b12 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.so+0xaa5820]  UTF8::unicode_length(char const*)+0x0
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/arzt/devel/SciView/hs_err_pid7018.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
```
[hs_err_pid7018.log](https://github.com/scenerygraphics/scenery/files/1952148/hs_err_pid7018.log)
[glxinfo.txt](https://github.com/scenerygraphics/scenery/files/1952152/glxinfo.txt)

**Comment by skalarproduktraum** [comment]
Thank you! This is actually helpful information, and the crash is not in scenery itself, but in the spirvcrossj (wrapped) library I'm using (https://github.com/scenerygraphics/spirvcrossj). Just for the fun of it, could you try running it with the Vulkan backend? (`-Dscenery.Renderer=VulkanRenderer`, you might need to `apt-get install libvulkan-dev`, your graphics driver is current enough though and doesn't need an update :+1: )

**Comment by maarzt** [comment]
Yes, I will try to run it with vulkan right now.

**Comment by maarzt** [comment]
It's the same situation with vulkan, sometime working sometimes failing. And also a crash with core dump: 
[hs_err_pid13529.log](https://github.com/scenerygraphics/scenery/files/1952646/hs_err_pid13529.log)

**Comment by skalarproduktraum** [comment]
Thanks! Just for the record, what's your exact distribution and kernel (`uname -a`)? I'm extending the tests in spirvcrossj at the moment so this maybe becomes traceable.

**Comment by skalarproduktraum** [comment]
The Vulkan issue you've posted has a different source, could you post the output of `vulkaninfo` as well? (Put it in a pastebin or sth please!)

**Comment by maarzt** [comment]
Ubuntu 16.04.1
Kernel #44~16.04.1-Ubuntu SMP

**Comment by maarzt** [comment]
[vulkaninfo.txt](https://github.com/scenerygraphics/scenery/files/1952711/vulkaninfo.txt)

**Comment by skalarproduktraum** [comment]
And while I'm investigating, could you also run the Vulkan version with `-Dscenery.LogLevel=debug` and post the console output along with the crash log in case it crashes?

**Comment by maarzt** [comment]
[console_output.txt](https://github.com/scenerygraphics/scenery/files/1952751/console_output.txt)
[hs_err_pid16232.log](https://github.com/scenerygraphics/scenery/files/1952752/hs_err_pid16232.log)

**Comment by skalarproduktraum** [comment]
The Vulkan issue your experiencing is a separate issue, filed as #151. Let's continue that discussion there.

For the other issue: Could you please clone the spirvcrossj repo, `git clone --recursive https://github.com/scenerygraphics/spirvcrossj.git`, and run `build.sh` there? After that please run `mvn test` repeatedly and see if it crashes at any point.

**Comment by maarzt** [comment]
I was able to build that `libspirvcrossj` thingy. I ran `mvn test` about 40 times, and it never failed.

**Comment by skalarproduktraum** [comment]
This issue has been fixed in 0.5.0 and was related to an incompatibility with the glslang version and and the Nvidia driver used. If anything along these lines still occurs, please reopen.
